### ISSUE TITLE
New Logo Design Offer as Open Source Contribution

### ISSUE THREAD

**Issue by anharismail** [ligi/FAST#103]
Hello Sir. I'm a UI/UX and Graphics Designer. I want propose a new logo for you. Would you mind if I propose new logo for your application as my Open Source Contribution?

Thanks before.

**Comment by ligi** [comment]
Yes I would be open to this - can you show me some of your work before you start?

**Comment by anharismail** [comment]
Yes, there's some of my portfolios

![cover](https://user-images.githubusercontent.com/37614260/38789047-1b8f6732-4127-11e8-97de-666df57bf83d.png)

![variations](https://user-images.githubusercontent.com/37614260/38789055-2ab08a70-4127-11e8-92f5-3fb302e6e129.png)

![variations](https://user-images.githubusercontent.com/37614260/38789066-3c92898c-4127-11e8-8e0d-44bda0fdbce2.png)

![cover](https://user-images.githubusercontent.com/37614260/38789075-501c1446-4127-11e8-96bf-d8578c0684ff.png)

**Comment by anharismail** [comment]
Hi Sir, can I start to design for my proposal logo?
